packages/malloy/src/lang/grammar/build_parser.js

```diff
@@ -18,7 +18,7 @@
 /* eslint-disable no-console */
 /* eslint-disable @typescript-eslint/no-var-requires */
 const { readFileSync, writeFileSync, existsSync, rmSync } = require("fs");
-const md5 = require("md5");
+const crypto = require('crypto');
```

Comment:
Unfortunately I don't think this is cross platform, the browser crypto implementation is different, and I'm not sure the browserify-crypto package will fit our needs.

Reply:
I guess I flagged the wrong instance of this, this is fine, it's the one's internal to the malloy library, like https://github.com/malloydata/malloy/pull/989/files#diff-99024c4735395f44ec7c632e0eb4feec5a3cf890db23a0cc9a4ca2d5f1e47dd2R71 that will be tricky cross platform.

Reply:
Yeah I knew what you meant.
Unfortunately it looks like dynamically loading libraries is async, as well as the browser crypto.subtle libraries, so that blocks making it easy to just use whichever is available at runtime without spreading asyc/await everywhere...